webapp/src/components/testing/TestSteps.vue

```diff
@@ -58,6 +58,34 @@
       </div>
     </div>
 
+    <div v-if="confusionMatrix && confusionMatrix.matrix.length > 0" class="p-4 mx-auto lg:w-1/2 h-full bg-white dark:bg-slate-950 rounded-md">
+      <h4 class="p-4 text-lg font-semibold text-slate-500 dark:text-slate-300">
+        Confusion Matrix
+      </h4>
+      <table class="min-w-full divide-y divide-slate-600 dark:divide-slate-400 text-center">
+        <thead>
+          <tr>
+            <th class="pl-6 py-3 text-xs font-medium text-gray-800 dark:text-gray-200 uppercase tracking-wider text-center border-r-gray-600 dark:border-r-gray-400 border-r-2 diagonal-header">                  
+              <span class="">Label \ Prediction</span>
```

Comment:
no style applied and already a standalone element
```suggestion
              Label \ Prediction
```